### PR TITLE
Refactor and standardized `node-aws-workers`

### PR DESCRIPTION
**Pull request by Kapilkoundinya**
@waynelee : r?

### CODE REVIEW

File: lib/config.js [hunk 1]
```diff
@@ -6,24 +6,24 @@
...
+'use strict';
 
 const path = require('path');
 const merge = require('lodash.merge');
 
 const LOCAL_CONFIG = 'local.config';
 
+// Configuration variables
 let config;
+let localConfig;
```
Comment (waynelee) [inline lib/config.js]: what was the reasoning for moving this to a variable global to the file?

File: lib/keys.js [hunk 2]
```diff
@@ -47,7 +46,7 @@ function cleanupKeys() {
     .then(() => {
       try {
         fs.unlinkSync(KEYFILE);
-      } catch (e) { }
+      } catch (e) { } // eslint-disable-line
```
Comment (waynelee) [inline lib/keys.js]: we should probably do something more intelligent here. can you add a TODO as well?

File: tools/cli.js [hunk 3]
```diff
@@ -78,28 +68,28 @@ function getInstance(instances, instanceName) {
   if (instanceName) {
     if (!isNaN(instanceName)) {
       if (instanceName >= instances.length) {
-        throw `Instance index '${instanceName}' does not exist`;
+        throw `Instance index '${instanceName}' does not exist`; // eslint-disable-line
```
Comment (waynelee) [inline tools/cli.js]: why `eslint-disable-line`?

Comment (Kapilkoundinya) [inline tools/cli.js]: Done

File: lib/keys.js (hunk 2 above)
Comment (Kapilkoundinya) [inline lib/keys.js]: Done

File: lib/config.js (hunk 1 above)
Comment (Kapilkoundinya) [inline lib/config.js]: For readability

**Comment by Kapilkoundinya** [comment]
@waynelee : r?

File: lib/config.js [hunk 4]
```diff
@@ -24,10 +24,7 @@ function getConfig() {
     return config;
   }
 
-  let cwd = process.cwd(); //__dirname;
-
   let defaultConfig = require(path.resolve('config', 'default'));
-  let localConfig;
```
Comment (waynelee) [inline lib/config.js]: `localConfig` is needed. I see that you moved it to be a file-global variable in a different commit, but it is only needed locally within this block.

Comment (Kapilkoundinya) [inline lib/config.js]: Done

**Pull request merged**